.env.local.example

```
@@ -17,6 +17,12 @@ NEXT_PUBLIC_COMPOSIO_USER_ID=user@example.com
 # NEXT_PUBLIC_MAPBOX_ACCESS_TOKEN=your_mapbox_public_token_here
 
 # AI Provider API Keys
+# Azure AI Foundry (Azure OpenAI)
+AZURE_RESOURCE_NAME=your_azure_resource_name_here
+AZURE_ENDPOINT=your_azure_endpoint_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AZURE_ENDPOINT key should go before the AZURE_RESOURCE_NAME key

(UnorderedKey)